src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -327,7 +327,12 @@
 
     private YesNo functionalTest;
 
-    private String caseName;
+    private String caseNameHmctsInternal;
```

Comment:
Might be worth moving these to some sort of object, as we are running low on the allowed fields in this class